test/e2e/k8sreporter/reporter.go

```diff
@@ -31,37 +30,29 @@ func New(clients *testclient.ClientSet, nameSpace string, reportPath string) *Ku
 	return &KubernetesReporter{clients: clients, reportPath: reportPath}
 }
 
-func (r *KubernetesReporter) SpecSuiteWillBegin(config config.GinkgoConfigType, summary *types.SuiteSummary) {
+func (r *KubernetesReporter) Report(report types.Report) {
+	r.mu.Lock()
```

Comment:
The copy/pasted reporter is now available as a library here https://github.com/openshift-kni/k8sreporter
It would make sense to consume it from here (note that the library doesn't pretend to be a ginkgo reporter, just a way to scrape the required informations, so it will make sense to have it wrapped inside this reporter)

Reply:
Done.